test/unit/vl-select.test.html

```diff
@@ -58,11 +58,20 @@
   <test-fixture id="vl-select-auto-dress-form-fixture">
     <template>
       <form>
-        <select id="select" is="vl-select" data-vl-select/>
+        <select id="select" is="vl-select" data-vl-select />
       </form>
     </template>
   </test-fixture>
 
+  <test-fixture id="multiple-selects-fixture">
+    <template>
+      <div>
+        <select id="select1" is="vl-select" data-vl-select data-vl-search-placeholder="placeholder 1"></select>
+        <select id="select2" is="vl-select" data-vl-select data-vl-search-placeholder="placeholder 2"></select>
+      </div>
```

Comment:
wij zijn toch nog altijd niet waar we willen zijn als we die teksten niet via proza component kunnen mee geven?

Reply:
Was me eigenlijk nog niet opgevallen, maar klopt.

Reply:
klopt... proza componenten lukken (nog) niet. Ik was teveel gefocust op de tekst in de placeholder die fout was. In de beschrijving van het ticket staat er inderdaad dat die van proza zouden moeten komen, dat had ik over het hoofd gezien.

Reply:
Ik heb net met Kenzo afgestemd dat de default teksten voorlopig voldoende zijn en dat er nog geen proza integratie nodig is.

Reply:
Ik had voorgesteld om de tekst logica van `vl-select-location` in `vl-map` eerst te verhuizen naar de `vl-select` en de globale i18n variabelen voor de dress call goed te zetten, dus ook defaults resetten in het geval dat er geen teksten overschreven worden.

Via een andere PR kon dan de slot logica uitbreiding toegevoegd worden. Als de default teksten voldoende zijn, is die 2e PR natuurlijk overbodig.